packages/server-core/src/githubapp/githubapp/githubapp.class.ts

```diff
@@ -0,0 +1,44 @@
+import { Id, NullableId, Params, ServiceMethods } from '@feathersjs/feathers'
+import { BadRequest, NotFound } from '@feathersjs/errors'
+import { Application } from '../../../declarations'
+import { getGitHubAppRepos, getGitRepoById } from './githubapp-helper'
+
+interface Data {}
+interface ServiceOptions {}
+
+export class GitHubAppService implements ServiceMethods<Data> {
+  app: Application
+  options: ServiceOptions
+  docs: any
+
+  constructor(options: ServiceOptions = {}, app: Application) {
+    this.options = options
+    this.app = app
+  }
+
+  async setup() {}
+
+  async find(): Promise<any> {
+    const repo = await getGitHubAppRepos()
+    return repo as any
+  }
+
+  async get(id: number): Promise<any> {
+    const repo = getGitRepoById(id)
+    return repo as any
+  }
+
+  async create(): Promise<any> {}
```

Comment:
lets remove these override functions if they are unused

Reply:
I tried to remove the unused override functions, but it shows this error:
```
Class 'GitHubAppService' incorrectly implements interface 'ServiceMethods<Data, Partial<Data>>'.
  Type 'GitHubAppService' is missing the following properties from type 'ServiceMethods<Data, Partial<Data>>': create, update, patch, removets(2420)
```

Reply:
ok dont worry about it then